docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -160,45 +160,75 @@ Then, you can use the `OidcClient.accessTokens` method to accept a Map of extra
 You might need to request that a specific set of scopes be associated with an issued access token.
 Use a dedicated `quarkus.oidc-client.scopes` list property, for example: `quarkus.oidc-client.scopes=email,phone`
 
+[[use-oidc-client-directly]]
 === Use OidcClient directly
 
-One can use `OidcClient` directly as follows:
+One can use `OidcClient` directly to acquire access tokens and set them as an HTTP `Authorization: Bearer` scheme value.
```

Comment:
```suggestion
You can use `OidcClient` directly to acquire access tokens and set them as an HTTP `Authorization: Bearer` scheme value.
```

Reply:
Question: Is "Authorization: Bearer" the actual name of the scheme value, with that exact formatting, colon included etc.?  

-

Reply:
@sheilamjones I'll try to simplify it, `Authorization:` is a header and the `Bearer` is the scheme.